### DISCUSSION TITLE
LDAP Authentication

### DISCUSSION THREAD

**Discussion by jviniciuspassos** [ciur/papermerge#352] [Q&A] [answered]
Does the LDAP Authentication from the pull request #146 work in the current version 2.0.0rc48?

**Comment by ciur** [comment, marked as answer]
No. I keep that pull request as reminder that LDAP authentication should be added.
The problem with that PR is that it adds LDAP as part of core, which should not be the case.
LDAP authentication will be added as external app (or plugin).
